src/IK/include/BiomechanicalAnalysis/IK/InverseKinematics.h

```diff
@@ -0,0 +1,99 @@
+/**
+ * @file InverseKinematic.h
+ * @authors Davide Gorbani <davide.gorbani@iit.it>
+ */
+
+#ifndef BIOMECHANICAL_ANALYSIS_INVERSE_KINEMATIC_H
+#define BIOMECHANICAL_ANALYSIS_INVERSE_KINEMATIC_H
+
+// iDynTree
+#include <iDynTree/KinDynComputations.h>
+
+// BipedalLocomotion
+#include <BipedalLocomotion/IK/QPInverseKinematics.h>
+#include <BipedalLocomotion/ParametersHandler/IParametersHandler.h>
+#include <BipedalLocomotion/System/VariablesHandler.h>
+#include <BipedalLocomotion/IK/SO3Task.h>
+
+namespace BiomechanicalAnalysis
+{
+
+namespace IK
+{
+
+class HumanIK
+{
+private:
+    // Integration time step
+    double m_dtIntegration;
+
+    // Joint positions and velocities
+    Eigen::VectorXd m_jointPositions;
+    Eigen::VectorXd m_jointVelocities;
+    Eigen::Vector3d m_basePosition;
+    manif::SE3Tangentd m_baseVelocity;
+    manif::SO3d m_baseOrientation;
+    Eigen::Vector3d m_baseAngularVelocity;
```

Comment:
I removed the `Eigen::Vector3d m_baseAngularVelocity` and converted `manif::SE3Tangentd m_baseVelocity` to an Eigen vector because in turned out to better work with the integrator.

Reply:
So in the end you are not using `manif` for defining the state. I don't know if @GiulioRomualdi has anything to comment about this point, for me it is fine!